ib_fabric.md

```diff
@@ -1,11 +1,28 @@
-# Infaband Fabric
+# InfiniBand Fabric
 
-The following information is for the Infaband (IB) Fabrics hosted within the lab.
+The following information is for the InfiniBand (IB) Fabrics hosted within the lab.
 
 ## Switch & Fabric
```

Comment:
Can we also document the switch management host name and IPv4 address in this section as well. Don't publish the management password, but this just makes it easier to know how to reach the switch.